packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -444,9 +449,24 @@ export class MultiSinkLogger extends TelemetryLogger {
 	constructor(
 		namespace?: string,
 		properties?: ITelemetryLoggerPropertyBags,
-		protected loggers: ITelemetryBaseLogger[] = [],
+		loggers: ITelemetryBaseLogger[] = [],
+		tryInheritProperties?: true,
 	) {
-		super(namespace, properties);
+		let realProperties = properties;
```

Comment:
this is just a more general form of something that was already in the debug logger. We look at the loggers that come in, and try to inherit their properties if they are a type we understand. This really only happens in our tests.

Reply:
we also do something similar in the child logger to help prevent nesting, and we had performance issues with deeply nested objects, and each layer would need to do a number of object allocations.